probfit/pdf.pyx

```diff
@@ -114,6 +114,94 @@ cdef class HistogramPdf:
             return 0.0
 
 
+cdef class _JohnsonSU:
+    """
+    Normalized JohnsonSU.
```

Comment:
Tiny suggestion: add in the docstring a reference for the JohnsonSU PDF. So something like:

```
Normalized JohnsonSU [1]_.
…

References
------------

.. [1] https://en.wikipedia.org/wiki/Johnson%27s_SU-distribution
```

Reply:
+1